documentation/16-start-stop/README.md

```diff
@@ -0,0 +1,27 @@
+Title: preStart, preStop, & postStop
```

Comment:
I feel like this should be rolled into the lifecycle page -- it's a bit disjointed otherwise and you can end up flipping back and forth between the two.